modules/virt-running-virtctl-ssh-command.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-accessing-vm-ssh.adoc
+
+:_content-type: PROCEDURE
+[id="virt-running-virtctl-ssh-command_{context}"]
+= Running the virtctl SSH command
+
+Run the `virtcl ssh` command to access a running virtual machine (VM).
+
+.Prerequisites
+
+* You added a static SSH public key to the VM before it booted for the first time.
```

Comment:
I doesn't have to be a static key. I would just mention that as a prerequisite an SSH pub key needs to be setup first.

Reply:
Also a prereq: `ssh` needs to be available on the machine executing `virtctl` (which likely it is most times anyway)

Reply:
4.13 supports only static keys.  This PR is for the current release.

4.14 supports dynamic keys. We will update the documentation.

Reply:
What kind of machine would NOT have ssh?

Reply:
>  What kind of machine would NOT have ssh?

Windows

Reply:
True, but we have notes all over the UI saying that the SSH fields are for Linux machines only and the RDP and sysprep settings are for Windows. I think that it might be overkill to mention SSH here. If someone is trying to use SSH on a Windows machine, I would assume that they know it is not a native command, no?

Reply:
In this place, the ssh client and virtctl has to be installed on the user's laptop. So even if the vm guest is linux, the user has to ensure to install the openssh client on windows.

Reply:
The SSH keys in a VM could have been configured without the help of CNV. `virtctl ssh` is agnostic to that. It is just an SSH client and does not care about how the keys were configured on the target.

Reply:
I added "SSH client" prerequisite.